crypto3/libs/blueprint/include/nil/blueprint/zkevm_bbf/exp.hpp

```diff
@@ -280,31 +282,31 @@ namespace nil {
                     }
 
                     for(std::size_t i = 0; i < max_rows; i++){
-                        allocate(header_selector[i], 0, i);
+                        allocate(header_selector[i], 0, i + 1);
```

Comment:
Same value `i+1` with (most probably) same meaning is used several times here. Please add a named variable and a comment:
```suggestion
                        size_t row_to_allocate = i + 1;  // we have to use adjusted index here because ...
                        allocate(header_selector[i], 0, i + 1);
```